unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/NavMap/NavmapView.cs

```diff
@@ -49,8 +49,7 @@ void Start()
             toastView.OnGotoClicked += () => navmapVisible.Set(false);
 
             MapRenderer.OnParcelClicked += TriggerToast;
-            MapRenderer.OnParcelHold += TriggerToast;
-            MapRenderer.OnParcelHoldCancel += () => { toastView.OnCloseClick(); };
+            MapRenderer.OnCursorFarFromParcel += () => { toastView.OnCloseClick(); };
```

Comment:
Beware, I think you forgot to unsubscribe from this method on `OnDestroy()`.

Reply:
Done with commit 21990120474e1cd08c47a37a5e7dc5ffd7cc7403